### PR TITLE
Added a docker build

### PR DESCRIPTION
**Pull request by jogster**
Updated existing makefile with a new docker build target

### CODE REVIEW

**Comment by janisz** [comment]
@jogster could you rebase to latest version?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM gliderlabs/alpine:3.2
```
Comment (kamilchm) [inline Dockerfile]: Why not official one https://hub.docker.com/_/alpine/ ?

File: README.md [hunk 2]
```diff
@@ -48,6 +49,14 @@ To build the binary:
 make build
 ```
 
+To build a docker container:
+
+```
+make build_docker
+```
+```
+make build
```
Comment (kamilchm) [inline README.md]: Is the `build_docker` not enough?

File: launch.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+[[ -n "$DEBUG" ]] && set -x
+
+if [ "$(ls -A /usr/local/share/ca-certificates)" ]; then
+  cat /usr/local/share/ca-certificates/* >> /etc/ssl/certs/ca-certificates.crt
```
Comment (kamilchm) [inline launch.sh]: What's the use case for this?

Comment (tsaridas) [inline launch.sh]: maybe to have http with the marathon-consul api ?

**Comment by tsaridas** [comment]
Maybe somebody can take over this pr since Brian seems to be busy. It would probably be best anyway if the owner or the repo also took care of the image @ the public docker registry

**Comment by janisz** [comment]
@tsaridas Good point. I think we can prepare Dockerfile and publish image in docker registry. Idea of packing statically linked binaries in Docker seems odd for me but I understand this is standart way of deploying applications for some systems (e.g., CoreOS).

If you are interested in this feature you can reopen #71.

**Comment by janisz** [comment]
@jogster @tsaridas I prepared `Dockerfile` to run marathon-consul. Binary need to be build before.
PR: #98

**Comment by tsaridas** [comment]
@jogster for convenience maybe you could push it to public docker repo ?

**Comment by janisz** [comment]
Fixed by #98

**Pull request closed without merging**